### PR TITLE
8 feature8 myflightslist

### LINKED ISSUE #8: feature#8-MyFlightsList
# REQUIREMENTS
Implement a flat list to render the item component.

# DESIGN

imagen

### PR DESCRIPTION
**Pull request by AmarilloVel**
# Description
 Created a component to render the flight items using flatlist component from react native.
modified the component flightitem, adding props as:
*  originLong
*  originShort
*  destLong
*  destShort
*  date
*  passengers
  
  ## Screenshots 

image

### CODE REVIEW

**Review by TonyHdez-Pi — APPROVED** [review]

File: flight-reservation/components/FlightListComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import { FlatList, View } from "react-native";
+import { FlightItem } from "../../components";
+import styles from "./styles";
+import React from "react";
+
+const data = [
```
Comment (wiltrouble) [inline flight-reservation/components/FlightListComponent/index.js]: Remove this data in a separate file

**Review by wiltrouble — CHANGES_REQUESTED** [review]

File: flight-reservation/components/FlightListComponent/index.js (hunk 1 above)
Comment (Astroyayo) [reply flight-reservation/components/FlightListComponent/index.js]: changes addressed

File: flight-reservation/components/FlightListComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { FlatList, View } from "react-native";
+import FlightItem from "../FlightItemComponent";
+import styles from "./styles";
+import React from "react";
+import DATA from "../../assets/data/DATA.json";
+
+const FlightList = () => {
+  const renderItem = ({ item }) => (
+    <View style={styles.itemContainer}>
```
Comment (wiltrouble) [inline flight-reservation/components/FlightListComponent/index.js]: Remove this View

**Review by wiltrouble — CHANGES_REQUESTED** [review]

File: flight-reservation/components/FlightListComponent/index.js (hunk 2 above)
Comment (Astroyayo) [reply flight-reservation/components/FlightListComponent/index.js]: addresed

**Review by wiltrouble — APPROVED** [review]
Looks good 👌

**Review by featlast — APPROVED** [review]
well done Nice Code !!!!

**Pull request merged**